### ISSUE TITLE
Adjust your reload segment

### ISSUE THREAD

**Issue by ShadowGamer3** [discordjs/guide#440]
As it stands at the time of post, the reload command guide works like a charm.  I love it, as it has been very useful.  However, it cannot reload the index.js file, therefore any changes made there require the bot to be rebooted manually.  You also have to reboot the bot anytime a new command is added, as the reload command can't see it yet until the bot is rebooted.  Please fix this, as there may be bots out there that, if rebooted, could (and probably would) terminate ongoing scripts, such as alarms, reminders, and intervals.  Being able to update the index(or bot).js file, or load up a new command without having to reboot the bot would be very useful for such bots (one of my own included).

I realize that this request is very similar to another that i made, but i felt it was different enough to warrant a new issue submission.

**Comment by tipakA** [comment]
There is no way to hot-reload main file. Since loading it again would execute all code in it, your bot would log in everytime you would do that, possibly creating copy of itself running in parallel with old instance. At this point it's literally easier to restart whole thing.

To avoid need to reload your main file, you shouldn't have anything in there that would require constant changes. I will eventually finish my event handler guide, which i can assume is what would help you achieve that.
